### ISSUE TITLE
Consider adding a TimeType

### ISSUE THREAD

**Issue by rapilabs** [schematics/schematics#413]
Consider adding a TimeType, independent of date, which maps to https://docs.python.org/3/library/datetime.html#time-objects

**Comment by bintoro** [comment]
Agreed. I wonder what to do about time zones, though. I'm going to guess most usage of `TimeType` is going to involve naive times, whereas the default behavior of the new `DateTimeType` is to produce a naive/aware datetime depending on if a time zone designator is present.

My first impression is that the `tzd` setting on `DateTimeType` could be directly transferred to `TimeType`, which kind of suggests the default behavior should also be the same between the two.
